src/osmf/js/osmf_flash.js

```diff
@@ -413,7 +414,7 @@ require("../../../html5-common/js/utils/constants.js");
       // Pause the video
       this.pause();
       // Reset the source
-      this.setVideoUrl('');
+      this.setVideoUrl('', '');
```

Comment:
Why do you need to pass an empty string here?  The parameter is not being used so it's okay if it's null or undefined (since '' is not more accurate anyways)